src/actions-on-google.ts

```diff
@@ -675,8 +709,16 @@ export class ActionsOnGoogle {
                         }
                     }
                 }
+                if (this.include && this.include.screenOut && data.screen_out
+                    && data.screen_out.format === 'HTML' && data.screen_out.data) {
```

Comment:
Does this pass `yarn lint`? Shouldn't this line have an extra indentation?

Reply:
Should `'HTML'` be a constant somewhere?

Reply:
linter and tests are passed for all my commits